WordPress/Classes/ViewRelated/NUX/LoginNavigationController.swift

```diff
@@ -0,0 +1,10 @@
+import UIKit
+import WordPressShared
+
+class LoginNavigationController: UINavigationController {
```

Comment:
I think I'll add in `RotationAwareNavigationViewController`, that's a good point.

`isWelcomeScreenVisible` is going to have to get butchered tho because I'm 99% sure the login flow is going to move to its own `UIWindow`.